src/mono/msbuild/common/MonoSharedBuild.props

```diff
@@ -0,0 +1,26 @@
+<Project>
```

Comment:
nit: I don't have a good suggestion, but I'd prefer the name of the file to not include `Mono` in it.

Reply:
As discussed offline, I renamed the file to `CommonMobileBuild.props`